week-3/B-array-some/exercise.js

```diff
@@ -6,17 +6,34 @@
   - Do not edit any of the existing code
 */
 
-var pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
+function checkNull(index){
+  //console.log(index);
+  return index==null || index == "";
```

Comment:
The exercise description is a little vague, but I think it means to only check for values that are `null`. That is slightly different from an empty string.